core/src/main/scala/org/broadinstitute/dsp/workbench/welder/package.scala

```diff
@@ -204,16 +197,22 @@ package object welder {
   def findFilesWithPattern(parent: Path, pattern: Regex): List[File] =
     parent.toFile.listFiles().filter(f => f.isFile && pattern.findFirstIn(f.getName).isDefined).toList
 
-  def getSourceUriForProvider(cloudProvider: CloudProvider, container: CloudStorageContainer, blob: CloudStorageBlob)(
-      implicit ev: Ask[IO, TraceId]
-  ): IO[SourceUri] =
-    ev.ask[TraceId].flatMap { traceId =>
-      cloudProvider match {
-        case CloudProvider.Gcp => IO(GsPath(container.asGcsBucket, blob.asGcs))
-        case CloudProvider.Azure => IO(AzurePath(container.asAzureCloudContainer, blob.asAzure))
-        case CloudProvider.None => IO.raiseError(InvalidSourceURIException(traceId, "Cannot get sourceURI with no cloud provider", Map.empty))
-      }
-    }
+//  def getSourceUriForProvider(cloudProvider: CloudProvider, container: CloudStorageContainer, blob: CloudStorageBlob)(
```

Comment:
Seems like this helper is probably not needed since we don't transform a `CloudStorageContainer` +`CloudStorageBlob` + `CloudProvider` into a `SourceUri` now, and use the the new types instead?

Reply:
ah yea...will clean up